tests/ui/unit/filters.tests.js

```diff
@@ -1,3 +1,7 @@
+import angular from 'angular';
+
+const { inject } = window;
```

Comment:
These help tell ESLint that the variables are expected to come from globals. The alternative would be to list them under `globals` in the ESLint config, but this seems more explicit.